source/release-notes.txt

```diff
@@ -10,6 +10,34 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.40.3
+----------------
+
+*Released October 11, 2023*
+
+New Features:
+
+- After creating a $search index you are now redirected to the search 
+  indexes modal (:issue:`COMPASS-7247`).
+- Field names are now autocompleted when defining indexes. (:issue:`COMPASS-7174`).
+- Syntax errors are now highlighted when defining indexes. (:issue:`COMPASS-7246`).
+- Improved AI feedback experience (:issue:`COMPASS-7211`, :issue:`COMPASS-7251`).
```

Comment:
(suggestion)
Include a link to the AI content.
"For more information on the Compass's Generative AI natural language query feature, see ... " Or something like this.